### ISSUE TITLE
Error on kill mobs

### ISSUE THREAD

**Issue by FabioZumbi12** [Rocologo/MobHunting#380]
This is the error (only the relevant part):

```
Could not pass event EntityDeathEvent to MobHunting v5.5.6
org.bukkit.event.EventException: null
...
...

Caused by: java.lang.NullPointerException
        at one.lindegaard.MobHunting.MobHuntingManager.onMobDeath(MobHuntingManager.java:1844) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor531.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_151]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_151]
        at us.Myles.PWP.TransparentListeners.PerWorldPluginLoader$1.execute(PerWorldPluginLoader.java:120) ~[?:?]
        ... 27 more
```

**Comment by Rocologo** [comment]
Please test build 600 or newer
https://fractal.lindegaard.one:8181/job/MobHunting/

**Comment by Rocologo** [comment]
Should be fixed in v5.5.7

**Comment by FabioZumbi12** [comment]
Hi,
I think no, the error still:

```
Could not pass event EntityDeathEvent to MobHunting v5.5.7
...
Caused by: java.lang.NullPointerException        at one.lindegaard.MobHunting.MobHuntingManager.onMobDeath(MobHuntingManager.java:1843) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor184.invoke(Unknown Source) ~[?:?]        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_151]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_151]        at us.Myles.PWP.TransparentListeners.PerWorldPluginLoader$1.execute(PerWorldPluginLoader.java:120) ~[?:?]
        ... 27 more
```
